app.vue

```diff
@@ -135,6 +135,9 @@ else {
         <TheDomainForm v-model:editing="editing" :domain="domain" />
       </div>
       <template v-if="!editing && domain">
+        <span v-if="(crux && crux.redirected) || (lighthouse && lighthouse.redirected)" class="text-gray-400">
+            Showing results for {{ crux?.domain }} {{ lighthouse?.domain }}
+          </span>
```

Comment:
I think I'd be very happy not to display this additional line here but just display the actual domain shown in the domain form above.

Reply:
one of reasons did it like this was to keep interaction as simple as possible, this is alsoohow pagespeed had done it, but i hear you. let me know..